src/test/rpc/Tx_test.cpp

```diff
@@ -528,6 +547,29 @@ class Tx_test : public beast::unit_test::suite
                 client.status.ok(), client.reply);
         };
 
+        auto grpcAccountTx = [&grpcPort](
+                                 uint256 const& id,
+                                 auto binary,
```

Comment:
I think you missed this generic param when you fixed the other two. `auto` -> `bool`